.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+- repo: https://github.com/asottile/pyupgrade
```

Comment:
pyupgrade isn't strictly necessary here, but it's a nice check to make sure you're using up-to-date syntax.